Assets/Mirror/Runtime/Transport/Transport.cs

```diff
@@ -94,6 +94,17 @@ public virtual void ClientConnect(Uri uri)
 
         #region Server
 
+
+        /// <summary>
+        /// Retrieves the address of this server.
+        /// Useful for network discovery
+        /// </summary>
+        /// <returns>the url at which this server can be reached</returns>
+        public virtual Uri ServerUri()
```

Comment:
just make it abstract. it's too easy to miss otherwise. this way people get a compile error before even running into any exceptions